src/App.tsx

```diff
@@ -9,56 +9,102 @@ import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
 import { Loader } from './components/Loader';
 
+// Context
+import { Provider } from './components/Context';
+
+// Types
+
+import { Post } from './types/Post';
+import { User } from './types/User';
+
 export const App: React.FC = () => {
-  return (
-    <main className="section">
-      <div className="container">
-        <div className="tile is-ancestor">
-          <div className="tile is-parent">
-            <div className="tile is-child box is-success">
-              <div className="block">
-                <UserSelector />
-              </div>
+  // State
```

Comment:
The same remove comments. It is clear that this is state, because you use useState hook